ipxe/menu.ipxe

```diff
@@ -0,0 +1,135 @@
+#!ipxe
+
+# dhcp should be already done by boot script
+#dhcp
+
+#set variables
+set menu-timeout 30000
+
+:start
+menu iPXE boot menu for openqa.opensuse.org 
+
+item --gap --             ------------------------- Operating systems ------------------------------
+item --gap --             ------ (You will be prompted to edit cmdline after selecting an os) ------
+item --key l leapS1       (L)Boot openSUSE Leap15.3 from download.opensuse.org / ttyS1 (http)
+item --key e leapS2       (E)Boot openSUSE Leap15.3 from download.opensuse.org / ttyS2 (http)
```

Comment:
why 15.3 which is EOL ?

Reply:
I just kept it, because I reused the existing menu partly.

Reply:
removed the 15.3 entries and added an autoyast entry